integrations/azure/.port/resources/blueprints.json

```diff
@@ -0,0 +1,447 @@
+          ],
+          "enumColors": {
+            "unavailable": "red",
+            "available": "green"
+          }
+        },
+        "statusOfSecondary": {
+          "title": "Status of Secondary",
+          "type": "string",
+          "enum": [
+                "available",
+                "unavailable"
+          ],
+          "enumColors": {
+            "unavailable": "red",
+            "available": "green"
+          }
+        },
+        "tags": {
+          "title": "Tags",
+          "type": "object"
+        },
+        "allowBlobPublicAccess": {
+          "title": "Allow Blob Public Access",
+          "type": "boolean"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "resourceGroup": {
+        "target": "resourceGroup",
+        "title": "Resource Group",
+        "required": true,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "storageContainer",
+    "description": "This blueprint represents an Azure Storage Container in our software catalog",
+    "title": "Storage Container",
+    "icon": "S3",
+    "schema": {
+      "properties": {
+        "publicAccess": {
+          "title": "Public Access",
+          "type": "string"
+        },
+        "hasImmutabilityPolicy": {
+          "title": "Has Immutability Policy",
+          "type": "boolean"
+        },
+        "hasLegalHold": {
+          "title": "Has Legal Hold",
+          "type": "boolean"
+        },
+        "deleted": {
+          "title": "Deleted",
+          "type": "boolean"
+        },
+        "deletedTime": {
+          "title": "Deleted Time",
+          "type": "string"
+        },
+        "remainingRetentionDays": {
+          "title": "Remaining Retention Days",
+          "type": "number"
+        },
+        "leaseStatus": {
+          "title": "Lease Status",
+          "type": "string"
+        },
+        "leaseState": {
+          "title": "Lease State",
+          "type": "string"
+        },
+        "defaultEncryptionScope": {
+          "title": "Default Encryption Scope",
+          "type": "string"
+        },
+        "version": {
+          "title": "Version",
+          "type": "string"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "storageAccount": {
+        "target": "storageAccount",
+        "title": "Storage Account",
+        "required": true,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "virtualMachine",
+    "description": "This blueprint represents an Azure Virtual Machine in our software catalog",
+    "title": "Virtual Machine",
+    "icon": "Azure",
+    "schema": {
+      "properties": {
+        "location": {
+          "title": "Location",
+          "type": "string"
+        },
+        "provisioningState": {
+          "title": "Provisioning State",
```

Comment:
Can this be an ENUM?

Reply:
doesn't have a DataModel for set of possible values :( https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/get?tabs=HTTP